pkg/apis/componentconfig/edgecore/v1alpha2/types.go

```diff
@@ -297,6 +297,7 @@ type EdgeHub struct {
 	// Optional if quic is configured
 	WebSocket *EdgeHubWebSocket `json:"websocket,omitempty"`
 	// Token indicates the priority of joining the cluster for the edge
+	// Deprecated: will be removed after release 1.14, will not be saved in configuration file
```

Comment:
```suggestion
	// Deprecated: will be removed in future release, will not be saved in configuration file
```
Need more discussion for removing in which one release :)

Reply:
yes, fixed